packages/utils/src/errors/packages.ts

```diff
@@ -3,6 +3,7 @@ export enum PACKAGES {
   APOLLO_CLIENT = '@8base/apollo-client',
   APOLLO_LINKS = '@8base/apollo-links',
   APP_PROVIDER = '@8base/app-provider',
+  REACT_AUTH = '@8base/react-auth',
```

Comment:
@vorobeez I think we should remove `@8base/react-native-auth0-auth-client` from this list

Reply:
Fixed